src/components/v5/frame/ColonyHome/partials/ReputationChart/ReputationChart.tsx

```diff
@@ -1,149 +1,87 @@
-import { Id } from '@colony/colony-js';
-import clsx from 'clsx';
-import React, { useState } from 'react';
+import { type PieTooltipProps, ResponsivePie } from '@nivo/pie';
+import React, { type FC } from 'react';
 import { defineMessages } from 'react-intl';
 
-import { Action } from '~constants/actions.ts';
-import { useActionSidebarContext } from '~context/ActionSidebarContext/ActionSidebarContext.ts';
 import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import Numeral from '~shared/Numeral/Numeral.tsx';
 import { notNull } from '~utils/arrays/index.ts';
 import { formatText } from '~utils/intl.ts';
-import { ACTION_TYPE_FIELD_NAME } from '~v5/common/ActionSidebar/consts.ts';
-import TeamReputationSummaryRow from '~v5/common/TeamReputationSummary/partials/TeamReputationSummaryRow.tsx';
-import WidgetBox from '~v5/common/WidgetBox/index.ts';
-import EmptyWidgetState from '~v5/common/WidgetBox/partials/EmptyWidgetState.tsx';
-import DonutChart from '~v5/shared/DonutChart/index.ts';
-import { type ChartData } from '~v5/shared/DonutChart/types.ts';
 
-import { summaryLegendColor } from './consts.ts';
-import { getTeamReputationChartData } from './utils.ts';
+import { pieChartConfig } from './consts.ts';
+import Legend from './partials/Legend.tsx';
+import LegendItem from './partials/LegendItem.tsx';
+import NoTeamsWithReputation from './partials/NoTeamsWithReputation.tsx';
+import { type ReputationChartDataItem } from './types.ts';
+import {
+  getTeamReputationChartData,
+  isThereReputationInDomains,
+} from './utils.ts';
 
 const displayName = 'v5.frame.ColonyHome.ReputationChart';
 
 const MSG = defineMessages({
-  legendNoTeams: {
-    id: `${displayName}.legendNoTeams`,
-    defaultMessage: 'There are no teams with reputation in the Colony',
-  },
-  legendNoTeamsAction: {
-    id: `${displayName}.legendNoTeamsAction`,
-    defaultMessage: 'Create a new action',
+  reputationTitle: {
+    id: `${displayName}.influenceTitle`,
+    defaultMessage: 'Influence by team',
   },
 });
 
+const TemporaryTooltip: FC<PieTooltipProps<ReputationChartDataItem>> = ({
+  datum: { label, value },
+}) => {
+  return (
+    <div className="bg-base-black p-2 text-base-white">
+      {label}
+      (<Numeral value={value.toFixed(1)} />
+      %)
+    </div>
+  );
+};
+
```

Comment:
Once #3044 gets merged, we can use this component https://github.com/JoinColony/colonyCDapp/blob/97b19407c4eb85ad1fd950c307d327fb90386c36/src/components/v5/frame/ColonyHome/partials/TotalInOutBalance/partials/ChartCustomTooltip.tsx